docs/_deploying/heroku.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Deploying to Heroku
+order: 0
+slug: heroku
+lang: en
+layout: tutorial
+permalink: /deploying/heroku
```

Comment:
The URL will be `/deploying/heroku` unless there is a better suggestion.

Reply:
I prefer `/deployments/heroku` or `/deployments-heroku`

Reply:
Awesome, this aligns well with some feedback from @shaydewael to use "Deployments" for the header section. I've now updated the URL to be `/deployments/heroku`

Fixed in f628553